### DISCUSSION TITLE
Renaming an existing relationship

### DISCUSSION THREAD

**Discussion by hertilizer** [Budibase/budibase#4395] [Help]
![image](https://user-images.githubusercontent.com/64743964/153052511-2dfc182c-f7a0-425b-b8fe-d689c71df042.png)
There doesn't seem to be a way to rename a relationship that you have already created. It automatically locks all of the fields when trying to edit a relationship. I am not talking about the error it gives me at the bottom of the pop up. I am trying to edit the name field, not the column name in other field.

**Comment by gzqn** [comment]
i deleted th relationship and recreated it using the some name in the other table, If you have data in the stored in the relationship field this would be lost though. if you are concerned about UI play with label in design.

**Reply by hertilizer** [reply]
Yes that's what I did as well, just thought there would be a better way of doing this. Got the UI covered in the Design Tab but just wanted an easy way to change this.

**Comment by hotemetoot** [comment]
Anyone found a way to do this? Would like to know how this works as well
